lib/db.js

```diff
@@ -434,6 +434,7 @@ module.exports = function (
             return bufferize({
               id: item.id,
               sessionToken: item.sessionTokenId,
+              lastConnectedAt: item.lastAccessTime,
```

Comment:
I was going to suggest `lastAccessTime` for consistency with the session data, but I really don't feel strongly about